python/cuml/test/test_nearest_neighbors.py

```diff
@@ -408,22 +416,42 @@ def test_nearest_neighbors_sparse(nrows, ncols,
 
     a = cp.sparse.random(nrows, ncols, format='csr', density=density,
                          random_state=32)
+    b = cp.sparse.random(nrows, ncols, format='csr', density=density,
+                         random_state=33)
 
-    logger.set_level(logger.level_info)
-    nn = cuKNN(metric=metric, n_neighbors=n_neighbors, algorithm="brute",
+    if metric == 'jaccard':
+        a = a.astype('bool').astype('float32')
+        b = b.astype('bool').astype('float32')
+
+    nn = cuKNN(metric=metric, p=2.0, n_neighbors=n_neighbors,
+               algorithm="brute", output_type="numpy",
                verbose=logger.level_debug,
```

Comment:
Can you remove `verbose=logger.level_debug`? This will print the debug output to every test log thats run in CI. Only really need that when there is a problem.